scripts/install/usr/libexec/cortx-motr/motr-monitor

```diff
@@ -44,5 +43,6 @@ while true; do
 		echo "restarting $1"
 		/usr/libexec/cortx-motr/motr-server "m0d-$1" &
 		PID=$!
+                /opt/seagate/cortx/motr/libexec/core_logrotate.sh "m0d-$1" "$PID" &
```

Comment:
Do we need to send "m0d-$1" and "$PID" as arguments? core_logrorate.sh script is not accepting these.

Reply:
we can use it to optimize in later

Reply:
No need to pass PID to core_logrotate.sh, as we will be handling duplicate PIDs using core filename format. No need to pass unnecessary/unused args.

Reply:
incorporated